template/src/screens/LoginScreen/index.ts

```diff
@@ -1 +1 @@
-export * from './component';
+export * from './LoginScreen';
```

Comment:
I like seeing these changes! Do we need to update the hygen generator so that when someone creates new component or screens that it doesn't add the files this way and thus stays consistent?

Reply:
yes we do 😄good call out